indico/modules/rb_new/util.py

```diff
@@ -104,3 +106,9 @@ def get_linked_object(type_, id_):
                         SessionBlock.session.has(db.and_(~Session.is_deleted,
                                                          Session.event.has(is_deleted=False))))
                 .first())
+
+
+def is_booking_start_within_grace_period(start_dt, user, allow_admin=False):
+    if allow_admin and rb_is_admin(user):
+        return True
+    return start_dt >= datetime.now() - timedelta(hours=1)
```

Comment:
this will break if the DEFAULT_TIMEZONE does not match the server's timezone. use utc and convert to the server's tz